src/client/components/Modal/Modal.component.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import './Modal.styles.css';
+import PropTypes from 'prop-types';
+import Button from '../Button/Button.component';
+import addedToCart from '../../assets/images/added_to_cart.png';
+
+export default function Modal({
+  modalIsOpen,
+  message,
+  cornerCloseButton,
+  primaryButtonLabel,
+  showConfirmIcon,
+  buttonFunction,
+  secondaryButtonFunction,
+  secondaryButtonLabel,
+}) {
+  return (
+    <div className="modal-background">
+      <div className="modal-box">
+        {cornerCloseButton && (
+          <button
+            onClick={() => modalIsOpen(false)}
```

Comment:
The usual format for naming event handlers like this would be `onModalClose`, that way the consumer can write code like this:
`<Modal onModalClose={() => setModalOpen(false)} />` 

If you send a boolean and you only ever send false, it's fine to not send anything.